packages/gateway/src/manager.ts

```diff
@@ -392,14 +378,9 @@ export function createGatewayManager(options: CreateGatewayManagerOptions): Gate
           },
           events: options.events ?? {},
           logger: this.logger,
-          requestIdentify: async () => {
-            await gateway.identify(shardId)
-          },
+          requestIdentify: async () => await gateway.requestIdentify(shardId),
           shardIsReady: async () => {
```

Comment:
Should we rename the method from `shardIsReady` to `shardIdentified` or `shardIsIdentified`?